dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -35,8 +35,8 @@ class VERSION(enum.Enum):
 class VERSION_MAPPING:
 
   class NIGHTLY(enum.Enum):
-    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-nightly-cp310-cp310-linux_x86_64.whl"
-    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-nightly-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0.dev-cp310-cp310-linux_x86_64.whl"
```

Comment:
it's not nightly anymore?

Reply:
Not nightly any more due to the issue here https://github.com/pytorch/xla/issues/7697. If you are using the [nightly](https://pantheon.corp.google.com/storage/browser/pytorch-xla-releases/wheels/cuda/12.1?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%257B_22k_22_3A_22_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22torch_xla-nightly-cp310-cp310-linux_x86_64.whl_5C_22_22_2C_22s_22_3Atrue%257D%255D%22,%22s%22:%5B(%22i%22:%22objectListDisplayFields%2FtimeCreated%22,%22s%22:%221%22),(%22i%22:%22displayName%22,%22s%22:%220%22)%5D))&e=13802955&inv=1&invt=AbZ6sg&mods=allow_workbench_image_override&project=tpu-pytorch-releases&prefix=&forceOnObjectsSortingFiltering=true), it's actually an old build from 08/13.